components/tyk-dev-portal/values.yaml

```diff
@@ -127,9 +127,13 @@ imagePullSecrets: []
 service:
   # type corresponds to the service type of Tyk Developer Portal Service.
   type: ClusterIP
+  
   # The port at which Tyk Developer Portal service can be found
   port: 3001
 
+  # the name given to this service port
+  name: ""
```

Comment:
i would suggest renaming this as `portName` or something else because `name` might reminds users configuring Service Name instead of service's port name